recipes/clair3/meta.yaml

```diff
@@ -1,32 +1,42 @@
 {% set name = "clair3" %}
-{% set version = "0.1.10" %}
+{% set version = "0.1.11" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: http://www.bio8.cs.hku.hk/clair3/bioconda/Clair3_v{{ version }}.zip
-  sha256: 0c222a7a8b56feba18958e8daa8b3001e9f303be0358feea268c713dea003c2c
-
-
+  sha256: e54667809b8b814dd456811dce0a6f53855d39502d8cc70eb1af3619f5e92d9e
 
 build:
   number: 0
-  noarch: generic
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cffi
+    - bzip2
+    - xz
+    - zlib
+    - python=3.6.10
```

Comment:
Is the strict pin for python needed?

Reply:
Hi! @[bgruening](https://github.com/bgruening), 

Yes, we need to execute the python script in MakeFile.

Also, I found the CI-built stopped in test_linux for almost 6 hours. I have an old [PR](https://github.com/bioconda/bioconda-recipes/pull/34034) early but failed multiple times, any suggestions on that, thanks!

Reply:
Using Python is ok, but why do you need this exact version (3.6.10)? Can you remove it? Also do you need this as run-dependency? I only see failed OSX tests, Linux seems to be fine.

Reply:
Yes, tensorflow2.2.0 only supports 3.6.10 for python3.6. Python needs to be installed in run-dependency as I need to compile CFFI using ffibuilder in python.
CI-built works now, thanks for your help!

Reply:
Yes, as tensorflow2.2.0 only supports 3.6.10 for python3.6. Python needs to be installed in run-dependency as I need to compile CFFI using ffibuilder in python.
CI-built works now, thanks for your help!